hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/PrefixManagerImpl.java

```diff
@@ -64,13 +64,7 @@ public class PrefixManagerImpl implements PrefixManager {
   // In-memory prefix tree to optimize ACL evaluation
   private RadixTree<OmPrefixInfo> prefixTree;
 
-  // TODO: This isRatisEnabled check will be removed as part of HDDS-1909,
-  //  where we integrate both HA and Non-HA code.
-  private boolean isRatisEnabled;
-
-  public PrefixManagerImpl(OzoneManager ozoneManager, OMMetadataManager metadataManager,
-      boolean isRatisEnabled) {
-    this.isRatisEnabled = isRatisEnabled;
+  public PrefixManagerImpl(OzoneManager ozoneManager, OMMetadataManager metadataManager) {
```

Comment:
OzoneManager instantiates a PrefixManagerImpl with isRatisEnabled = true and
OmSnapshotManager instantiates a PrefixManagerImpl with isRatisEnabled = false.

I would suggest to restore PrefixManagerImpl, and rename isRatisEnabled to persistentToRocksDB, because it looks like OmSnapshotManager repurposed it and which is unrelated to Ratis.

Reply:
Restored, now the tests are unchanged compared to `master`.  However, I kept the name `isRatisEnabled`, someone more familiar with snapshots can rename it separately.